### PR TITLE
Add tag to project and search card

### PR DESCRIPTION
**Pull request by VeerpalBrar**
Display the first tag on each project card and search card.

### CODE REVIEW

File: client/src/components/ProjectCard.vue [hunk 1]
```diff
@@ -57,11 +56,18 @@ export default {
...
   ],
   computed: {
     updateLink: function() {
       return "/connect/projects/" + this.project_id + "/update";
+    },
+    tag: function(){
+      if (this.tags.length > 0) {
+        return this.tags[0].tag_name
+      }
+      return ''
```
Comment (VeerpalBrar) [inline client/src/components/ProjectCard.vue]: handles the edge case where a project might not have any tags.

File: client/src/views/Search.vue [hunk 2]
```diff
@@ -87,11 +83,9 @@ export default {
     projectExists(row, col) {
       return this.getProject(row, col) != null;
     },
-      hasNoProjects() {
-          if (this.project_list == 0) {
-              return true;
-          }
-      },
+    hasNoProjects() {
+      return this.project_list.length == 0  && this.show
```
Comment (VeerpalBrar) [inline client/src/views/Search.vue]: Let's only show the no projects found message after the user clicks the search button to search for tags.

File: client/src/components/ProjectCard.vue (hunk 1 above)
Comment (itsninaricci29) [reply client/src/components/ProjectCard.vue]: Looks good! Thank you for doing that. I wasnt sure on how to display it

File: client/src/views/Search.vue (hunk 2 above)
Comment (itsninaricci29) [reply client/src/views/Search.vue]: Oh okk

**Pull request merged**